ledger/sandbox-classic/src/test/suite/scala/platform/sandbox/stores/ledger/sql/SqlLedgerSpecAppendOnly.scala

```diff
@@ -211,23 +210,6 @@ final class SqlLedgerSpecAppendOnly
         ledger.currentHealth() should be(Healthy)
       }
     }
-
-    /** Workaround test for asserting that PostgreSQL asynchronous commits are disabled in
```

Comment:
Why are we getting rid of this test completely rather than replacing it with something equivalent? This would merit a TODO, wouldn't it?

Reply:
I am not so sure. 
What happened here is a major change in how we configure asynchronous commit. 
Previously the general asynchronous commit configuration was overridden in a couple of cases (this included), so we still configure synchronous commit even if configuration was asynchronous commits.
Currently we do not care at all at all the places about asynchronous commit configuration (everything will be just default eg sync commit), except from one dedicated place: the parallel ingestion.
With this change it seemed to me unnecessary and very hard to test something like this.

Reply:
What do you think?

Reply:
Sounds good